go.mod

```diff
@@ -1,15 +1,18 @@
 module github.com/zclconf/go-cty
 
 require (
+	github.com/apparentlymart/go-cidr v1.0.1
 	github.com/apparentlymart/go-textseg v1.0.0
-	github.com/golang/protobuf v1.1.0 // indirect
+	github.com/bmatcuk/doublestar v1.1.5
 	github.com/google/go-cmp v0.3.1
-	github.com/kr/pretty v0.1.0 // indirect
+	github.com/google/uuid v1.1.1
+	github.com/hashicorp/go-uuid v1.0.2
+	github.com/hashicorp/hcl/v2 v2.3.0
 	github.com/kylelemons/godebug v0.0.0-20170820004349-d65d576e9348
+	github.com/mitchellh/go-homedir v1.1.0
 	github.com/vmihailenco/msgpack v3.3.3+incompatible
-	golang.org/x/net v0.0.0-20180811021610-c39426892332 // indirect
-	golang.org/x/sync v0.0.0-20180314180146-1d60e4601c6f // indirect
-	golang.org/x/text v0.3.0
-	google.golang.org/appengine v1.1.0 // indirect
-	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
+	golang.org/x/crypto v0.0.0-20200117160349-530e935923ad
+	golang.org/x/text v0.3.2
 )
+
+go 1.13
```

Comment:
Ah; go added this for me but I'm not sure this is helpful here; should we drop it ?

Reply:
I believe this is used just to add an extra note to error messages prompting that they might be caused by the requirement of a new Go version. For Terraform's sake this codebase is still targeting Go 1.12 compatibility, so perhaps we could explicitly set this to `go 1.12`.

I think Go 1.13 will not try to update it if it's already there, but that this behavior is just to encourage adding a version specifier to improve the UX for future Go language changes.

Reply:
Got it; setting this to 1.12 for now 🙂